docs/_includes/generated/checkpointing_configuration.html

```diff
@@ -30,7 +30,7 @@
         <tr>
             <td><h5>state.backend.local-recovery</h5></td>
             <td style="word-wrap: break-word;">false</td>
-            <td></td>
+            <td>This option configures local recovery for this state backend. By default, local recovery is deactivated. Local recovery currently only covers keyed state backends. Some state backends may not support local recovery and ignore this option.</td>
```

Comment:
If we can clearly describe **which** state backend does not support local recovery. It would be better.

Reply:
Thanks for the review! It's a good suggestion. I have clarified it in the latest commit.